hoomd/md/methods.py

```diff
@@ -132,6 +115,31 @@ def _attach(self):
                                  "")
         super()._attach()
 
+
+    def thermalize_extra_dof(self, seed):
+        """Set the extra degrees of freedom to random values.
+
+        Args:
+            seed (int): Random number seed
+
+        Some integration methods, such as `NVT` and `NPT` employ extra degrees
+        of freedom for the thermostat and/or barostat. `thermalize_extra_dof`
+        sets these to random values. This method performs no operation for
+        integration methods that lack internal degrees of freedom.
```

Comment:
Since no classes have this method where it does nothing, we no longer need this statement.

Reply:
Done.